bgpd/bgp_bmp.c

```diff
@@ -1084,35 +1384,36 @@ static struct stream *bmp_withdraw(const struct prefix *p,
 	return s;
 }
 
+/* sends a bmp monitoring message using the given information on the bmp session
+ *
+ * if uptime is (time_t)(-1L) then do not include the timestamp in the message
+ */
```

Comment:
actually, I saw the undefined timer issue in bmp mirroring messages.

Reply:
ok. I mean that if you added the test to see if time == -1, then you know there are some cases where the time value is not set. I suppose this is something that should be done later in a separate pr.  Hence the need to write it somewhere in doc/developer/bmp.rst for instance.